crates/bevy_ecs/src/schedule/executor/mod.rs

```diff
@@ -77,18 +77,18 @@ impl SystemSchedule {
     }
 }
 
-/// Instructs the executor to call [`apply_buffers`](crate::system::System::apply_buffers)
-/// on the systems that have run but not applied their buffers.
+/// Instructs the executor to call [`apply_deferred`](crate::system::System::apply_deferred)
```

Comment:
```suggestion
/// Instructs the executor to call [`System::apply_deferred`](crate::system::System::apply_deferred)
```

The current formatting might be confusing since these two functions have the same name.